app/service/data_svc.py

```diff
@@ -65,6 +65,27 @@ def __init__(self, dao):
                 for fact in source['facts']:
                     fact['source_id'] = source_id
                     await self.create_fact(**fact)
+                relationships = source.get('relationships')
+                if relationships:
+                    for relationship in relationships.values():
+                        await self.create_relationship(relationship)
+
+    async def create_relationship(self, values):
+        return await self.dao.create('core_fact_relationships', dict(value1=values['value1'],
+                                                                     relationship=values['relationship'],
+                                                                     value2=values['value2']))
+
+    @staticmethod
+    def get_ability_relationship(operation, ability_id, relationship_type):
```

Comment:
1) this function should live under the """ VIEW """ section in this module
2) add a docstring
3) I don't like that this function accepts an operation object-- this makes it appear too tightly coupled with that object. it seems like a better name would be get_operation_relationships. and if that is the case, I think it should live on the operation_svc or the planning_svc (probably the latter)